definitions/utils/nxdl_desc2rst.py

```diff
@@ -382,7 +382,9 @@ def getDocFromNode(ns, node, retval=None):
             text = lines[0][indent:]
         for line in lines[1:]:
             if not len(line[:indent].strip()) == 0:
```

Comment:
Why not just remove this if statement?

Reply:
To modify the code as little as possible. I do not know what the importance of this piece of code is, only that it is not working.

Reply:
ok!